parquet/README.md

```diff
@@ -1,33 +1,33 @@
+This folder contains sample parquet files that work within the Deephaven console. 
 
 ## Table of contents
 
-- `alltypes_dictionary.parquet`: Parquet file using int32, int 64, int 96 timestamps
-- `alltypes_plain.parquet`: Parquet file using int32, int 64, int 96 timestamps
-- `alltypes_plain.snappy.parquet`: Parquet file using int32, int 64, int 96 timestamps with snappy compression
-- `customer.impala.GZIP.parquet`: Parquet file with GZIP compression
-- `customer.impala.NONE.parquet`: Parquet file with no compression
-- `customer.impala.SNAPPY.parquet`: Parquet file with snappy compression
-- `flow.snappy.parquet`: Parquet file doubles and with snappy compression
-- `monthlyProductSales.parquet`: Parquet file with whitespace
-- `nation.dict-malformed.parquet`: Non-Standard Parquet file
-- `nation.impala.SNAPPY.parquet`: Parquet file with int32 and binary, snappy compression
-- `nation.impala.GZIP.parquet`: Parquet file with int32 and binary, GZIP compression
-- `nation.impala.NONE.parquet`: Parquet file with int32 and binary, no compression
-- `nested_lists.snappy.parquet`: Nested Parquet file with snappy compression
-- `nested_maps.snappy.parquet`: Nested Parquet map file with snappy compression
-- `nonnullable.impala.parquet`: Nested Parquet file with maps and no nulls
-- `nullable.impala.parquet`: Nested Parquet file with maps and nulls
-- `nulls.snappy.parquet`: Nested Parquet file with maps and nulls, snappy compression
-- `repeated_no_annotation.parquet`: Nested Parquet file with repeat values
-- `stock_simulated.parquet`: Sample stock data in Parquet format
-- `taxi.parquet`: Parquet data with millis timestamps
-- `test_datapage_v2.snappy.parquet`: Nested Parquet list file with snappy compression
+- `taxi.parquet`: uncompressed parquet file
 
+## Fields in the `taxi.parquet` file
 
+`station_data.csv` has 17 fields:
+
+  - **int64** VendorID
+  - **int64**  tpep_pickup_datetime (TIMESTAMP(MICROS,false))
+  - **int64**  tpep_dropoff_datetime (TIMESTAMP(MICROS,false))
+  - **int64**  passenger_count
+  - **double**  trip_distance
+  - **int64**  RatecodeID
+  - **binary**  store_and_fwd_flag (STRING)
+  - **int64**  PULocationID
+  - **int64**  DOLocationID
+  - **int64**  payment_type
+  - **double**  fare_amount
+  - **double**  extra
+  - **double**  mta_tax
+  - **double**  tip_amount
+  - **double**  tolls_amount
+  - **double**  improvement_surcharge
+  - **double**  total_amount
 
 # Source and License
 
-This data was built from data sets publicly available.  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
+This data was built from data sets publicly available on [Microsoft Azure](https://azure.microsoft.com/en-us/services/open-datasets/catalog/nyc-taxi-limousine-commission-yellow-taxi-trip-records/). It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```

Comment:
A few more comments/questions that I don't have a better place to put them:
1) `parquet/.DS_Store` still exists.  See:  https://github.com/deephaven/examples/tree/parquetFiles/parquet
2) Where did the testing parquet files get moved to?